components/form-builder/util.ts

```diff
@@ -259,3 +259,31 @@ export const isVaultDelivery = (deliveryOption: DeliveryOption | undefined) => {
 export const isEmailDelivery = (deliveryOption: DeliveryOption | undefined) => {
   return !!(deliveryOption && deliveryOption.emailAddress);
 };
+
+export const escapeAngleBrackets = (value: string) => {
+  const openRegex = /<(?!\s)/g;
+  const closeRegex = /(?<!\s)>/g;
+  return value.replaceAll(openRegex, "< ").replaceAll(closeRegex, " >");
+};
+
+export const cleanInput = (input: any) => {
+  if (typeof input === "string") {
+    return escapeAngleBrackets(input);
+  }
+
+  if (typeof input === "object" && input !== null) {
+    const obj = Array.isArray(input) ? [...input] : { ...input };
+
+    for (const key in obj) {
+      if (typeof obj[key] === "object" && obj[key] !== null) {
+        obj[key] = cleanInput(obj[key]);
+      } else if (typeof obj[key] === "string") {
+        obj[key] = escapeAngleBrackets(obj[key]);
+      }
+    }
+
+    return obj;
+  }
+
+  return input;
+};
```

Comment:
A recursive type can avoid the `any`. I think the code below should work and avoid the `in` operator which is flakey for some objects and arrays.

```suggestion
type Cleanable = string | Cleanable[] | { [key: string]: Cleanable } | unknown;

export const cleanInput = (input: Cleanable) => {
    if (typeof input === "string") {
        return escapeAngleBrackets(input);
    }
    if ( Array.isArray(input) ) {
        return input.map((elem) => cleanInput(elem));
    }
    if ( input instanceof Object ) {
        return Object.fromEntries(
            Object.entries(input).map(
                ([key, value]) => [key, cleanInput(value)]
            )
        )
    }
    return input;
};
};
```

Reply:
Thanks @dmethvin-gov you beat us to cleaning up the for this :)

That's a nice great improvement on the recursion code 🎉 .